lbrynet/core/Wallet.py

```diff
@@ -609,7 +609,21 @@ def update_metadata(self, new_metadata, old_metadata):
             meta_for_return[k] = new_metadata[k]
         return defer.succeed(Metadata(meta_for_return))
 
+    """
+    Claim a name, update if name already claimed by user
+    @param name: name to claim
 
+    @param bid: bid amount
+
+    @param m: metadata
+
+    @return: Deferred which returns a dict containing below items
+        txid - txid of the resulting transaction
+        nout - nout of the resulting claim
+        fee - transaction fee paid to make claim
+        claim_id -  claim id of the claim
+
+    """
```

Comment:
being pedantic, but I prefer the google style of docstrings, and most of lbrynet uses them.  

The google style is:

```Required short single line explanation.

Optional longer explanation

Args:
    name_of_argument: description

Returns: a description of what the function returns
```

More examples: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html